ols/constants.py

```diff
@@ -117,7 +117,7 @@ class GenericLLMParameters:
 # And if we set a very high value, then there is risk of discarding all the chunks,
 # as there won't be perfect matching chunk.
 # Range: 0 to 1
-RAG_SIMILARITY_CUTOFF = 0.3
+RAG_SIMILARITY_CUTOFF = 0.635
```

Comment:
@romartin can you plz add a comment, as to WDYT that 0.635 is better then 0.3, and improves quality